wandb/sdk/internal/sender.py

```diff
@@ -1070,7 +1080,6 @@ def _start_run_threads(self, file_dir: Optional[str] = None) -> None:
             self._api,
             self._run.run_id,
             self._run.start_time.ToMicroseconds() / 1e6,
-            timeout=self._settings._file_stream_timeout_seconds,
```

Comment:
why??? this fix is here to avoid hangs! don't remove it!

Reply:
I've accidentally made a bunch of reverts here. I will audit all the changes in the PR right now.